### ISSUE TITLE
Service org.erikreider.swayosd has access denied

### ISSUE THREAD

**Issue by mrpingo** [ErikReider/SwayOSD#73]
Hello!

just run swayosd-libinput-backend with the following error:

`Error: org.freedesktop.DBus.Error.AccessDenied: Connection ":1.50" is not allowed to own the service "org.erikreider.swayosd" due to security policies in the configuration file`

How I can modify the policy to match the security policy?

Thanks in advance.

**Comment by migue07juegos** [comment]
Have you tried [this](https://github.com/ErikReider/SwayOSD/discussions/30#discussioncomment-6472198)? I had the same issue and it worked for me.

**Comment by mrpingo** [comment]
My.. bad.. so sorry for making you all waste your time if I did. It was written crystal clear in the README and also at the issues tables.

I went crazy (in my country: "like a chicken without head") looking in the wrong places (DBus related topics). Also I found bad config for DBus looking for this :S.

As a suggestion to avoid people like me asking silly questions would be adding this error to the README part that mentions the prefix. I couldn't relate the issue with the /usr prefix configuration.

Thanks for everything!
